jdbc-adapter/README.md

```diff
@@ -88,34 +88,41 @@ DROP VIRTUAL SCHEMA hive CASCADE;
 
-**Typical Optional Parameters:**
+Property                    | Value
+--------------------------- | -----------
+**CONNECTION_NAME**         | Name of the connection created with `CREATE CONNECTION` which contains the JDBC connection string, the username and password. If you defined this property then it is not allowed to set CONNECTION_STRING, USERNAME and PASSWORD. We recommend using this property to ensure that the password will not be shown in the logfiles.
+**CONNECTION_STRING**       | The JDBC connection string. Only required if CONNECTION_NAME is not set.
+**USERNAME**                | Username for authentication. Only required if CONNECTION_NAME is not set.
+**PASSWORD**                | Password for authentication. Only required if CONNECTION_NAME is not set.
+
+
+**Common Optional Properties:**
 
-Parameter                   | Value
+Property                    | Value
 --------------------------- | -----------
-**CATALOG_NAME**            | The name of the remote jdbc catalog. This is usually case-sensitive, depending on the dialect. It depends on the dialect whether you have to specify this or not. Usually you have to specify it if the data source JDBC driver supports the concepts of catalogs.
-**SCHEMA_NAME**             | The name of the remote jdbc schema. This is usually case-sensitive, depending on the dialect. It depends on the dialect whether you have to specify this or not. Usually you have to specify it if the data source JDBC driver supports the concepts of schemas.
-**USERNAME**                | Username for authentication. Can only be set if CONNECTION_NAME is not set.
-**PASSWORD**                | Password for authentication. Can only be set if CONNECTION_NAME is not set.
+**CATALOG_NAME**            | The name of the remote JDBC catalog. This is usually case-sensitive, depending on the dialect. It depends on the dialect whether you have to specify this or not. Usually you have to specify it if the data source JDBC driver supports the concepts of catalogs.
+**SCHEMA_NAME**             | The name of the remote JDBC schema. This is usually case-sensitive, depending on the dialect. It depends on the dialect whether you have to specify this or not. Usually you have to specify it if the data source JDBC driver supports the concepts of schemas.
 **TABLE_FILTER**            | A comma-separated list of table names (case sensitive). Only these tables will be available as virtual tables, other tables are ignored. Use this if you don't want to have all remote tables in your virtual schema.
 
-
 **Advanced Optional Properties:**
 
-Parameter                   | Value
+Property                    | Value
 --------------------------- | -----------
 **IMPORT_FROM_EXA**         | Only relevant if your data source is EXASOL. Either 'TRUE' or 'FALSE' (default). If true, IMPORT FROM EXA will be used for the pushdown instead of IMPORT FROM JDBC. You have to define EXA_CONNECTION_STRING if this property is true.
```

Comment:
Would be cool if you could change all the SQL statements and property names to monospace, i.e. IMPORT FROM EXA/JDBC and EXA_CONNECTION_STRING here.

Please review at least the Adapter Properties section, I think there are many properties which we could convert to monospace. Then we have this finally consistent. You can make a separate pull request for this documentation fixes.